include/rocksdb/status.h

```diff
@@ -151,6 +151,21 @@ class Status {
     return state_.get();
   }
 
+  // Override this status with another, unless this status is already non-ok.
+  // Returns *this. In other words, the result of `a.Update(b).Update(c)` is
+  // non-ok (and `a` modified as such) iff any input was non-ok, with
+  // left-most taking precedence as far as the details.
+  Status& Update(Status&& s) {
```

Comment:
Just wondering if `UpdateIfOk` would be better. It wasn't obvious to me that this is a no op call for non-ok status until I saw the comment.

Reply:
Better than my name. Done.